swagger.yaml

```diff
@@ -3161,6 +3161,7 @@ definitions:
       - ntp-synchronizer
       - fio-perf-check
       - container-image-availability
+      - domains-resolver
```

Comment:
```suggestion
      - domain-resolution
```

Reply:
I don't mind changing to `-resolution` but are you sure about changing it to singular?